### DISCUSSION TITLE
Upnp and Airplay bridges SSL lib error on new lyrionmusicserver container images

### DISCUSSION THREAD

**Discussion by docgalaxyblock** [LMS-Community/slimserver-platforms#72] [General]
Hi, I just tried updating to the new container but the bridge plugins complain about SSL libaries:

```
Starting Squeeze2raop: /config/cache/InstalledPlugins/Plugins/RaopBridge/Bin/squeeze2raop-linux-x86_64 -Z -I -f /config/logs/raopbridge.log -x /config/prefs/raopbridge.xml
[10:05:53.073] main:1583 Starting squeeze2raop version: v1.7.7 (Apr  2 2024 @ 16:11:41)

[10:05:53.076] Start:1305 Cannot load SSL libraries
[10:05:53.076] main:1683 Cannot start, exiting
```

```
Starting Squeeze2upnp: /config/cache/InstalledPlugins/Plugins/UPnPBridge/Bin/squeeze2upnp-linux-x86_64 -Z -I -f /config/logs/upnpbridge.log -x /config/prefs/upnpbridge.xml
[10:12:58.520] main:1831 Starting squeeze2upnp version: v3.3.9 (Apr  5 2024 @ 22:09:43)
[10:12:58.522] Start:1510 Cannot load SSL libraries
[10:12:58.522] main:1864 Cannot start, exiting
```

Tested on `:stable` and  `:dev/:9.1.0`

**Comment by michaelherger** [comment]
Did you try to use a statically linked version of those files? I believe you can select them from the plugins' respective settings pages.

**Comment by docgalaxyblock** [comment]
Year the static one works

**Comment by docgalaxyblock** [comment]
Maybe leave it open for a while that others see it faster?
Also as a reminder that the dynamic ones are broken?

**Comment by michaelherger** [comment]
The brokenness would probably be something for @philippe44. And most users would rather go to forums.slimdevices.com than come to GitHub. I prefer to keep the issues list clean... But I'll convert to a discussion. I hope that's fine with you.
